views/stats.ejs

```diff
@@ -1663,7 +1663,7 @@ twitterDescription += '. Click to view more stats!'
                 <% } %>
             <% } %>
             <p class='stat-raw-values'><small><i>
-                Please note, that this section is still in the works, since Hypixel doesn't support Dungeons in their public API yet.<br>
+                Please note,that this section is still in the works, since Hypixel doesn't support Dungeons in their public API yet.<br>
```

Comment:
```suggestion
                Please note that this section is still in the works, since Hypixel doesn't support Dungeons in their public API yet.<br>
```
Comma shouldn't be there